master/internal/experiment.go

```diff
@@ -214,7 +214,7 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 				return err
 			}
 
-			if !j.QPos.Equals(decimal.Zero) {
+			if j.QPos.GreaterThanOrEqual(decimal.Zero) {
```

Comment:
the jobs already get the zero position by default so maybe we don't need to send this msg if the position is at `0`

Reply:
yes I'd say this should be strictly greater than 0

Reply:
I was thinking what if the receiving actor hooks onto this message for whatever other reason or if more information ends up being carried with this message later? wouldn't it be more expected for the msg to be sent considering that we won't get 0 q position as a default value?